content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -0,0 +1,164 @@
+- **Note:** When the driver is created, MDM value for initContainers in driver CR is set by the operator from mdm attributes in the driver configuration file, 
+  config.yaml. An example of config.yaml is below in this document. Do not set MDM value for initContainers in the driver CR file manually.
+- **Note:** To use an sdc-binary module from customer ftp site:
+  - Create a secret, sdc-repo-secret.yaml to contain the credentials for the private repo. To generate the base64 encoding of a credential:
+ ```yaml
+      echo -n <credential>| base64 -i
+``` 
+   secret sample to use:
+ ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: sdc-repo-creds
+        namespace: vxflexos
+      type: Opaque
+      data:
+        # set username to the base64 encoded username, sdc default is
+          username: <username in base64>
+        # set password to the base64 encoded password, sdc default is
+          password: <password in base64>
+```  
+  - Create secret for FTP side by using the command `kubectl create -f sdc-repo-secret.yaml`.
+  - Optionally, enable sdc monitor by uncommenting the section for sidecar in manifest yaml. Please note: 
+    - **If using sidecar**, you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. 
+    - **If not using sidecar**, please leave this commented out -- otherwise, the empty fields will cause errors.
+##### Example CR:  [samples/storage_csm_powerflex_v240.yaml](https://github.com/dell/csm-operator/blob/main/samples/storage_csm_powerflex_v240.yaml)
+```yaml
+    sideCars:
+    # sdc-monitor is disabled by default, due to high CPU usage 
+      - name: sdc-monitor
+        enabled: false
+        image: dellemc/sdc:3.6
+        envs:
+        - name: HOST_PID
+          value: "1"
+        - name: MDM
+          value: "10.xx.xx.xx,10.xx.xx.xx" #provide MDM value
+```  
+ *Note:* Please comment the sdc-monitor sidecar section if you are not using it. Blank values for MDM will result in error.
+
+#### Manual SDC Deployment
+
+For detailed PowerFlex installation procedure, see the _Dell PowerFlex Deployment Guide_. Install the PowerFlex SDC using this procedure:
+
+**Steps**
+
+1. Download the PowerFlex SDC from [Dell Online support](https://www.dell.com/support). The filename is EMC-ScaleIO-sdc-*.rpm, where * is the SDC name corresponding to the PowerFlex installation version.
+2. Export the shell variable _MDM_IP_ in a comma-separated list using `export MDM_IP=xx.xxx.xx.xx,xx.xxx.xx.xx`, where xxx represents the actual IP address in your environment. This list contains the IP addresses of the MDMs.
+3. Install the SDC per the _Dell PowerFlex Deployment Guide_:
+    - For Red Hat Enterprise Linux and CentOS, run `rpm -iv ./EMC-ScaleIO-sdc-*.x86_64.rpm`, where * is the SDC name corresponding to the PowerFlex installation version.
```

Comment:
Should we update this line due to the new init container supporting more OS?

Reply:
What additional oses do we support? Where do I find that information?

Reply:
rhel 7.9 and 8.4 will be supported by init container when this releases. 
To avoid mentioning specific OS, I would change this line: "For Red Hat Enterprise Linux and CentOS..." to "For environments that require manual SDC install..." 
So we don't need to update it when SDC team changes their support matrix

Reply:
Updated it to say, "For environments using RPM..."